pkg/kubectl/resource/builder.go

```diff
@@ -607,14 +619,21 @@ func (b *Builder) visitByResource() *Result {
 		isSingleItemImplied = len(b.resourceTuples) == 1
 	}
 
+	result := &Result{
+		singleItemImplied:  isSingleItemImplied,
+		targetsSingleItems: true,
```

Comment:
> I'm not sure what you're arguing for. I don't want each line to have 40 characters that are exactly the same that someone will cut and paste incorrectly.

I'd prefer a `.with` construct, but I won't stick on it.  There already be dragons here.